### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by OlenaArshan**
Current value and profit/loss calculator. Please review tasks from homework 2.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -6,17 +6,36 @@
...
+            string userInput = Console.ReadLine();
+            int nominal = int.Parse(userInput ?? "0");
 
-    class Program
-    {
-        static void Main(string[] args)
-        {
+            Console.WriteLine("Input the trade price of the deal:");
+            userInput = Console.ReadLine();
+            double tradePrice = double.Parse(userInput ?? "0");
```
Comment (alberttashu) [inline Program.cs]: you can remove this condition - `userInput ?? "0"`; 
If user won't input anything in the console, Console.ReadLine would return `string.Empty`;
So a better way to do such condition would be 
`tradePrice = string.IsNullOrEmpty(userInput) ? 0 : double.Parse(userInput)`

File: Program.cs [hunk 2]
```diff
@@ -6,17 +6,36 @@
...
 
+            Console.WriteLine("Input the transaction type:");
+            userInput = Console.ReadLine();
+            TransactionType trcType = (TransactionType)Enum.Parse(typeof(TransactionType), userInput ?? "0", true);
```
Comment (alberttashu) [inline Program.cs]: this condition doesn't make sense here - `userInput ?? "0"`
default value "0" won't be parsed to TransactionType anyway

File: Program.cs (hunk 1 above)
Comment (alberttashu) [reply Program.cs]: and please consider changing all occurences of `userInput ?? "0"`

**Review by alberttashu — CHANGES_REQUESTED** [review]

**Comment by OlenaArshan** [comment]
Thank you for comments, I made changes.

**Pull request merged**